wifi-host-driver/WiFi_Host_Driver/inc/whd_types.h

```diff
@@ -189,7 +189,8 @@ typedef void* whd_m2m_t;
  * taken into account along with this during buffer pool creation. Also buffer pools needs alignment with
  * cache size of the platform for better performance
  */
-#define WHD_LINK_MTU            (WHD_PAYLOAD_MTU + WHD_PHYSICAL_HEADER)
+// #define WHD_LINK_MTU            (WHD_PAYLOAD_MTU + WHD_PHYSICAL_HEADER)
+#define WHD_LINK_MTU            (1600)
```

Comment:
This is a concern for me. Changing this value may make things work for SPI, but I would assume has impact on everything else. I don't know what the right thing to do is, but I am confident this is not it. We may need to get our Wi-Fi experts in to talk about this.

In the source WHD driver, this used to be a fixed 1500. The change to WHD_PAYLOAD_MTU + WHD_PHYSICAL_HEADER seems to have occurred recently.

Reply:
@ifyall - I dug up the conversation about WHD_LINK_MTU.  The original definition adds up to 1544 bytes, but @yiding saw what appeared to be a message 1592 bytes long when updating the CLM data.  However, I set this back to the original definition and I'm not seeing that problem.
I took this change out of whd_types.h and pushed that to this pull request.